### PR TITLE
RS-270: Improve MQTT storage article

### PR DESCRIPTION
**Pull request by AnthonyCvn**
- Reduce article to 3 main sections (instead of 4)
- Improve navigation (clearer section + anchors to jump to sections)
- Add links to other interesting blogs

### CODE REVIEW

**Comment by AnthonyCvn** [comment]
@atimin any suggestion on what can be improved?

File: blog/2024-05-17-mqtt-data-storage/index.mdx [hunk 1]
```diff
@@ -114,105 +116,93 @@ In addition, if you plan on performing analytics on your MQTT data, you will nee
...
-
-- [**Prometheus**](https://prometheus.io/): another widely used open-source monitoring system and time-series database designed for collecting metrics from various sources. It offers powerful querying capabilities, alerting functionalities, and easy integration with Grafana (an open-source visualization and monitoring tool).
+- [**ReductStore**](/): a specialized time-series database designed for storing MQTT data in the form of time-series blob data. It allows for storing unstructured data (blobs) along with their associated timestamps, making it suitable for scenarios where MQTT data includes large textual information, images, audio, videos, or other types of binary files.
 
-### ReductStore for time-series blob data
+- [**InfluxDB**](https://www.influxdata.com/): an open-source database that provides high-performance storage and retrieval of time-stamped data with a SQL-like query language. You can store numbers (integer or floating point values), boolean values, or text strings–with a limit of 64KB–that's 4000 times less than MQTT's payload capacity.
```
Comment (atimin) [inline blog/2024-05-17-mqtt-data-storage/index.mdx:150]: what about to remove all the external links?

File: blog/2024-05-17-mqtt-data-storage/index.mdx [hunk 2]
```diff
@@ -114,105 +116,93 @@ In addition, if you plan on performing analytics on your MQTT data, you will nee
...
 
-- [**Google Cloud Storage**](https://cloud.google.com/storage) is a globally distributed object storage service offered by Google Cloud Platform. They provide trustworthy and scalable databases for storing large amounts of blob data. They also provide a way to optimize cost and performance with different storage classes and pricing options.
+- [**Amazon S3**](https://aws.amazon.com/s3/): a widely used object storage service thanks to its global infrastructure and integration with other AWS services. They also provide features like lifecycle policies, versioning, and server-side encryption.
 
-- [**Amazon S3**](https://aws.amazon.com/s3/) is a widely used object storage service thanks to its global infrastructure and integration with other AWS services. They also provide features like lifecycle policies, versioning, and server-side encryption.
+- [**MinIO**](https://min.io/): an open-source object storage system that is compatible with Amazon S3 and provides high performance and scalability for storing large amounts of unstructured data. It is designed to be lightweight and easy to deploy on edge devices or in the cloud.
```
Comment (atimin) [inline blog/2024-05-17-mqtt-data-storage/index.mdx:188]: link to our benchmarks

**Review by atimin — CHANGES_REQUESTED** [review]
I like this version, see my suggestions. Additionally, we could disable the hiding navigation menu when a user scrolls the page down.

**Review by atimin — APPROVED** [review]
Prefect! Let's merge.

**Pull request merged**